js1/vstup-vystup-objekty/cvlekce/ockovani-objekty.md

```diff
@@ -13,13 +15,15 @@ Pokračujme v našem registračním systému na očkováni. Zatím se umíme už
    }
    ```
 1. Poté, co uživatel zadá všechny údaje, vypište do stránky objekt `person` v nějakém hezkém formátu a zkontrolujte, že obsahuje správné informace.
+1. Přidejte do objektu `person` údaj o tom, v jakém jazyce si uživatel přeje komunikovat. Zjistěte jej z objektu `window`.
 
 :::solution
 
 ```js
 const person = {
- name: prompt("Jake je vase jmeno a primeni?"),
+  name: prompt("Jake je vase jmeno a primeni?"),
   age: Number(prompt("Jaky je vas vek?")),
+  language: window.navigator.language,
 };
 ​
 document.body.innerHTML += "<p>Zadane jmeno je: " + person.name + ", vas vek je: " + person.age + ", boli ste zaregistrovany na ockovanie.</p>";
```

Comment:
Když už se zjišťuje jazyk, tak by bylo dobré ho i vypsat, aby si účastnice ověřily, že na to šly dobře.